tailwind.config.ts

```diff
@@ -43,7 +44,40 @@ const config = {
       },
     },
   },
-  plugins: [require("tailwindcss-animate")],
-} satisfies Config;
+  plugins: [
+    tailwindcssAnimate,
+    plugin(function ({ addUtilities, theme, e }) {
+      const newUtilities = {};
+      const colors = theme("colors");
+
+      const mergedColors = {
+        ...colors,
+        ...defaultTheme.colors,
+      };
+
+      Object.keys(mergedColors).forEach((colorName) => {
+        const value = mergedColors[colorName];
+        if (typeof value === 'string') {
+          const name = `.customshadow-${e(colorName)}`;
```

Comment:
better try this way if we need in multiple places:
https://tailwindcss.com/docs/box-shadow#customizing-your-theme

If it's one time use, https://tailwindcss.com/docs/box-shadow#arbitrary-values